src/clients/SpokePoolClient.ts

```diff
@@ -570,7 +581,7 @@ export class SpokePoolClient extends BaseAbstractClient {
         deposit.fromLiteChain = this.isOriginLiteChain(deposit);
         deposit.toLiteChain = this.isDestinationLiteChain(deposit);
 
-        if (deposit.outputToken === ZERO_ADDRESS) {
+        if (deposit.outputToken === ZERO_ADDRESS || deposit.outputToken === ZERO_BYTES) {
```

Comment:
@bmzig I think we'd benefit from a utility function here, a bit as we have with `isMessageEmpty()`.

Something like `isOutputTokenZero(deposit)`. wdyt?

Reply:
Done here https://github.com/across-protocol/sdk/pull/829/commits/2dd1bb0dec27fa8fd3f60e3f760cb99cffd5513f. I also reverted the `isMessageEmpty` change since we can probably get something better than what was done. Changing the `getRelayDataHash` is still pending the contract changes.